src/binding/python/Container.cpp

```diff
@@ -90,7 +90,7 @@ namespace detail
             std::forward< Args >( args ) ...
         );
 
-        cl.def( py::init<>() );
+        cl.def( py::init<Map const &>() );
```

Comment:
@ax3l Is this the correct way to stop exposing the (non-copy-)constructor in the Python interface?

Reply:
yes, you simply just define the copy constructor (but no other constructor) here.